djangoplicity/media/tasks.py

```diff
@@ -826,5 +826,5 @@ def image_observation_tagging_notification(pk):
         'New Observation image: {}'.format(url),
         '',
         settings.DEFAULT_FROM_EMAIL,
-        ['zidmani@gmail.com'],
+        settings.DEFAULT_MAIL_TAGGING,
```

Comment:
Again this is not a Django setting and would eventually cause errors if not set, this should be done only if the setting is set:
```python
if hasattr(settings, 'DEFAULT_MAIL_TAGGING'):
   send_mail ...
```